components/automate-ui/src/app/entities/clients/client-details.reducer.ts

```diff
@@ -47,6 +53,26 @@ export function clientDetailsEntityReducer(
         state
       ) as ClientDetailsEntityState;
 
+    case ClientActionTypes.RESETKEY:
+      return set(
+        GET_STATUS,
+        EntityStatus.loading,
+        clientDetailsEntityAdapter.removeAll(state)
+      ) as ClientDetailsEntityState;
```

Comment:
Remove redundant type cast; it is only needed on `pipe` calls, not `set` calls.
```suggestion
      );
```